src/main/scala/org/broadinstitute/dsde/firecloud/utils/PermissionsSupport.scala

```diff
@@ -49,7 +49,7 @@ trait PermissionsSupport {
     tryIsAdmin(userInfo) flatMap { isadmin =>
       if (!isadmin) {
         rawlsDAO.getWorkspace(workspaceNamespace, workspaceName)(userInfo.asInstanceOf[WithAccessToken]) map { ws =>
-          ws.accessLevel >= neededLevel
+          ws.accessLevel.getOrElse(WorkspaceAccessLevels.NoAccess) >= neededLevel
```

Comment:
Not 100% sure about this

Reply:
i think it's a fairly safe assumption to make, but a more idiomatic approach may be to use pattern matching:

```
          ws.accessLevel match {
            case Some(accessLevel) => accessLevel >= neededLevel
            case None => false
          }
```

pattern matching may also be useful in some other areas here too if you want to short-circuit any sort of evaluation that would be done by supplying a default in `getOrElse`. sorry i didn't mention this earlier when we were slacking!

Reply:
totally! thanks for the help!